va/va.h

```diff
@@ -2003,6 +2015,142 @@ typedef struct _VAEncMiscParameterParallelRateControl {
 } VAEncMiscParameterParallelRateControl;
 
 /**
+ * \brief MB partition modes and 1/2 1/4 motion search configuration
+ *
+ * Specifies MB partition modes that are disabled. Specifies Half-pel
+ * mode and Quarter-pel mode searching
+ */
+typedef struct _VAEncMiscParameterSubMbPartPel
+{
```

Comment:
Is this generic enough to cover all codecs? I don't think so, It rather specific to h264. 
Even the union name "inter_sub_mb_partition_mask" is so h264 specific.
If we really want to use this for all codecs in future, at least the "reserved" field should be extended with more size (so that we can add other fields in future without breaking the ABI)and  rename "inter_sub_mb_partition_mask" as "inter_partition_mask"

How about using a single 32bit Flag so that we can easily do the bit masking for all combinations?

Reply:
it's for H264, will move it into va_enc_h264.h and rename to  VAEncMiscParameterSubMbPartPelH264